torch/_library/utils.py

```diff
@@ -215,6 +215,67 @@ def zip_schema(
     return
 
 
+def normalize_args_kwargs(
```

Comment:
Wait, I've definitely seen this function somewhere else in the recent inductor typing PRs, or atleast a very similar function. The args and kwargs should be annotated with ParamSpec so they they properly pass the types of the args and don't disable argument checking.

The other piece of code I reviewed recently was a decorator too.